user-management/create_users.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import fire
+import json
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+parent_dir = os.path.dirname(current_dir)
+sys.path.append(parent_dir)
+
+from toolbox.get_access_token import get_access_token
+from toolbox.post_request import post_request
+
+
+def create_users(base_url, client_id, client_secret, input_file_path="./sample-files/create_users_sample.csv", email_verified=0):
+    try:
+        api_url = base_url + "/api/v1/users"
+        access_token = get_access_token(base_url, client_id, client_secret)
+
+        with open(input_file_path, newline='') as input_file:
+            users = csv.reader(input_file, delimiter=',')
+            for user in users:
+                new_user = {
+                    "email": user[0],
+                    "name": user[1],
+                    "password": user[2],
+                    "emailVerified": email_verified
+                }
+                response = post_request(api_url, access_token, new_user)
+
+                if "json" in response.headers["content-type"]:
+                    json_response = json.loads(response.text.encode("utf8"))
+                    print(json_response)
```

Comment:
and I believe the `response` object has a `response.json()` method

Reply:
ohh, is this for pretty-printing?
consider pprint - https://docs.python.org/3.10/library/pprint.html

Reply:
updated https://github.com/datasaur-ai/api-client-example/pull/22/commits/f3bd37e94e9b897c45756dca617505ec52180105